src/Microsoft.ML.CodeGen/Microsoft.ML.CodeGenerator.csproj

```diff
@@ -0,0 +1,119 @@
+    </None>
+    <None Update="Templates\Console\ModelInputClass.tt">
+      <Generator>TextTemplatingFilePreprocessor</Generator>
+      <LastGenOutput>ModelInputClass.cs</LastGenOutput>
+    </None>
+    <None Update="Templates\Console\ModelOutputClass.tt">
+      <Generator>TextTemplatingFilePreprocessor</Generator>
+      <LastGenOutput>ModelOutputClass.cs</LastGenOutput>
+    </None>
+    <None Update="Templates\Console\ModelProject.tt">
+      <Generator>TextTemplatingFilePreprocessor</Generator>
+      <LastGenOutput>ModelProject.cs</LastGenOutput>
+    </None>
+    <None Update="Templates\Console\PredictProgram.tt">
+      <Generator>TextTemplatingFilePreprocessor</Generator>
+      <LastGenOutput>PredictProgram.cs</LastGenOutput>
+    </None>
+    <None Update="Templates\Console\PredictProject.tt">
+      <Generator>TextTemplatingFilePreprocessor</Generator>
+      <LastGenOutput>PredictProject.cs</LastGenOutput>
+    </None>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Service Include="{508349b6-6b84-4df5-91f0-309beebad82d}" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Update="Templates\Console\ConsumeModel.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>ConsumeModel.tt</DependentUpon>
+    </Compile>
+    <Compile Update="Templates\Console\ModelBuilder.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>ModelBuilder.tt</DependentUpon>
+    </Compile>
+    <Compile Update="Templates\Console\ModelInputClass.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>ModelInputClass.tt</DependentUpon>
+    </Compile>
+    <Compile Update="Templates\Console\ModelOutputClass.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>ModelOutputClass.tt</DependentUpon>
+    </Compile>
+    <Compile Update="Templates\Console\ModelProject.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>ModelProject.tt</DependentUpon>
+    </Compile>
+    <Compile Update="Templates\Console\PredictProgram.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>PredictProgram.tt</DependentUpon>
+    </Compile>
+    <Compile Update="Templates\Console\PredictProject.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>PredictProject.tt</DependentUpon>
+    </Compile>
+    <Compile Update="Templates\Templates\Console\ModelInputClass.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>ModelInputClass.tt</DependentUpon>
+    </Compile>
+    <Compile Update="Templates\Templates\Console\ModelOutputClass.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>ModelOutputClass.tt</DependentUpon>
+    </Compile>
+  </ItemGroup>
+  
+  <Import Project="..\mlnet\mlnet.Build.props" />
```

Comment:
TODO: remove this line. (is used in feature branch) 

cc: @eerhardt

Reply:
Yes, let's remove this and the `mlnet.Build.props` file.

---
In reply to: [337701957](https://github.com/dotnet/machinelearning/pull/4365#discussion_r337701957) [](ancestors = 337701957)